zaza/openstack/charm_tests/ceph/rbd_mirror/tests.py

```diff
@@ -166,6 +184,18 @@ def setUpClass(cls):
         # get ready for multi-model Zaza
         cls.site_a_model = cls.site_b_model = zaza.model.get_juju_model()
         cls.site_b_app_suffix = '-b'
+        # Test if we have the needed Openstack applications.
+        try:
+            zaza.model.get_application(cls.cinder_ceph_app_name)
+            cls.with_cinder = True
+        except Exception:
```

Comment:
Please tighten up the exception. Which exception are you expecting when the application name is not present (`KeyError` or a custom exception).  This will silently set `cls.with_cinder` to `False` on a network error to the controller for example, and not fail in the test with a suitable error message.

Reply:
Done.